*: align description about pd-control with Chinese (#10014)
pingcap/docs: pd-control.md

### Download the installation package

> You can obtain the latest `pd-ctl` by downloading the TiDB server installation package. `pd-ctl` is included in the `ctl-{version}-linux-amd64.tar.gz` package.

Reviewer: ```suggestion
To obtain`pd-ctl` of the latest version, downloadthe TiDB server installation package. `pd-ctl` is included in the `ctl-{version}-linux-amd64.tar.gz` package.
```
